src/app/styles/components/_directions-form.scss

```diff
@@ -411,4 +405,60 @@
         }
 
     }
+
+    .directions-to {
+         &.isochrone-control {
+             display: none;
+         }
+
+         &.directions-text-input {
+             display: flex;
+         }
+     }
+
+     #isochrone-slider {
+        @include inputrange();
+        margin-right: 1rem;
+        width: 70%;
+    }
+
+    &.directions-destination-mode-within {
```

Comment:
Yeah, some classes were renamed in the prototype merge, for which I didn't refactor our base.